components/automate-chef-io/content/docs/install.md

```diff
@@ -156,8 +156,14 @@ Add the following to your config.toml:
 # [global.v1.external.elasticsearch.ssl]
 #  Specify either a root_cert or a root_cert_file
 #  root_cert = """$(cat </path/to/cert_file.crt>)"""
-#  root_cert_file = "</path/to/cert/file>"
 #  server_name = "<elasticsearch server name>"
+
+# Uncomment and fill out if using external elasticsearch that uses hostname-based routing/load balancing
```

Comment:
does this need to be added to the config docs page too?

Reply:
I can certainly go ahead and make the additions there too!

Reply:
on second thought, I see that there is no documentation for the external ES, I think that's more for if you're using the "built-in" ES cluster vs outsourcing it